packages/teleterm/src/ui/StatusBar/ShareFeedback/ShareFeedback.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { ButtonIcon, ButtonPrimary, Flex, Link, Text } from 'design';
+import Validation from 'shared/components/Validation';
+import { Close } from 'design/Icon';
+import { ShareFeedbackForm } from './ShareFeedbackForm';
+import { ShareFeedbackFormValues } from './types';
+
+interface ShareFeedbackProps {
+  onClose(): void;
+}
+
+export function ShareFeedback(props: ShareFeedbackProps) {
+  const [formValues, setFormValues] = useState<ShareFeedbackFormValues>({
+    feedback: '',
+    email: '',
```

Comment:
Could we pre-fill the email field with the username if the username looks like an email (`/^\S+@\S+$/`)?

Reply:
Ah yes, done